tbconnect/models.py

```diff
@@ -95,3 +95,23 @@ class TBCheck(ExportModelOperationsMixin("tb-check"), models.Model):
     language = models.CharField(
         max_length=3, choices=LANGUAGE_CHOICES, null=True, blank=True
     )
+
+
+class TBTest(ExportModelOperationsMixin("tb-test"), models.Model):
+    RESULT_POSITIVE = "positive"
+    RESULT_NEGATIVE = "negative"
+    RESULT_PENDING = "pending"
+    RESULT_CHOICES = (
+        (RESULT_POSITIVE, "Positive"),
+        (RESULT_NEGATIVE, "Negative"),
+        (RESULT_PENDING, "Pending"),
+    )
+
+    deduplication_id = models.CharField(max_length=255, default=uuid.uuid4, unique=True)
+    created_by = models.CharField(max_length=255, blank=True, default="")
+    msisdn = models.CharField(
+        max_length=255, validators=[za_phone_number], db_index=True
```

Comment:
I see we have an index on this field, but no way of querying/filtering by this field (same with the `timestamp` field). Are there plans for querying by those fields, or can we remove these indexes until we need them?

Reply:
I guess we can remove it for now.